topology_builder/src/main/java/com/yelp/pyleus/PyleusTopologyBuilder.java

```diff
@@ -141,7 +141,7 @@ public static IRichSpout handleKafkaSpout(
         }
 
         SpoutConfig config = new SpoutConfig(
-            new ZkHosts(zkHosts),
+            new ZkHosts(zkHosts, zkRoot),
```

Comment:
I suppoose this behavior is the one in https://github.com/apache/storm/blob/b2a8a77c3b307137527c706d0cd7635a6afe25bf/external/storm-kafka/src/jvm/storm/kafka/ZkHosts.java

However, I believe there is confusion about what is zkRoot (Root path in ZooKeeper to store consumer offsets, which defaults to /pyleus-kafka-offsets/<topology name>) and brokerZkPath in ZkHosts (which defaults to /kafka/brokers/). In my opinion this one should be a whole different option.

However, I have not deep knowledge of Kafka. Passing the ball to @ecanzonieri here.